modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -1,126 +1,132 @@
-* Deployed process definitions: The processes will continue to run using the definition created in the previous version of Bonita.
-* Process definition sources (`.bos` files): Migrate these by importing them into the new version of Bonita Studio.
-* +++<a id="bdm_redeploy">++++++</a>+++Business data model, and the business data database: if the migration path include version `7.0.0`,`7.2.0` or `7.2.4`, the Business data model must be redeployed after migration. You can pause the tenant before migration, as a tenant admin, so that you'll be able to redeploy the BDM on a paused tenant once migration is done, using  link:define-and-deploy-the-bdm[Define and deploy the BDM]). Otherwise, no action is required.
-* Custom connectors, actor filers, data types: These might continue to work in the new version, but should be tested, depending on your custom code.
-* Custom pages: These might continue to work in the new version, but should be tested depending on your custom code.
-* Custom reports: These might continue to work in the new version, but should be tested depending on your custom code.
-* REST API Extensions: These might continue to work in the new version, but should be tested depending on your custom code.
+* Bonita 7.0 only supports JRE version 7. If updating to Bonita 7.0 to 7.4.x, please update your JRE to version 7.
+* Bonita 7.5 only supports JRE version 8. If updating to Bonita 7.5 to Bonita 2021.1-0811, please update your JRE to version 8.
+* Bonita 2021.2 only supports JRE version 11. If updating to Bonita 2021.2 or greater, please update your JRE to version 11.
 
-When the script has finished,
-you need to complete the migration by unzipping and configuring a bundle for the new version.
-See  <<migrate,Migrate your platform>> for step-by-step instructions.
+For more information, go to the Customer Service Center and read the Support Guide and Supported Environment Matrix for Server.
+
+[#rdbms_requirements]
 
-== Constraints
+=== Database
+* The migration script supports MySQL, Postgres, Oracle, and Microsoft SQL Server. *There is no migration for h2 database*.
+* The target Bonita version may not support the *version of the database* used with your source Bonita version. You may then need to update the version of your database prior to running the migration tool. To make sure:
+ ** Please check the xref:hardware-and-software-requirements.adoc[database requirements].
+ ** If you need to update your database, make sure to apply all the xref:database-configuration.adoc#specific_database_configuration[RDBMS customisations required by Bonita] when setting up the new version.
+* If you have added *custom indexes* to certain tables in the engine database, you must *remove them before lauching* the update procedure. If you do not remove these indexes, the update procedure will not complete.
```

Comment:
will not complete. =>  will not be complete.

Reply:
fixed